ui/app/templates/components/attributes-section.hbs

```diff
@@ -3,17 +3,23 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 
-{{#each-in this.attributes as |key value|}}
-  {{#if (is-object value)}}
-    <tr data-test-attributes-section>
-      <td data-test-heading class="is-subheading" colspan="2">
-        {{#if this.prefix}}<span class="is-faded" data-test-prefix>{{this.prefix}}.</span>{{/if}}{{key}}
-      </td>
-    </tr>
-    <AttributesSection @prefix={{if this.prefix (concat this.prefix "." key) key}} @attributes={{value}}
-    @key={{key}} @value={{value}} @editable={{@editable}} @onKVSave={{@onKVSave}} @copyable={{@copyable}} />
-  {{else}}
-    <MetadataKv @prefix={{this.prefix}}
-    @key={{key}} @value={{value}} @editable={{@editable}} @onKVSave={{@onKVSave}} @copyable={{@copyable}} />
-  {{/if}}
+{{#each this.attributes.files as |file|}}
+  <MetadataKv
+    @prefix={{file.prefix}}
+    @key={{file.name}}
+    @value={{file.variable.value}}
+    @editable={{@editable}}
+    @onKVSave={{@onKVSave}}
+    @copyable={{@copyable}}
+  />
+{{/each}}
+{{#each-in this.attributes.children as |key value|}}
+  <AttributesSection
+    @attributes={{value}}
+    @key={{key}}
+    @value={{value}}
+    @editable={{@editable}}
+    @onKVSave={{@onKVSave}}
+    @copyable={{@copyable}}
+  />
```

Comment:
Two notable changes here:
- We no longer show the "key-prefix-only" blue headers like we used to. These were `colspan="2"`/full-width rows that both broke table accessibility conventions and made the page taller than it needed to be (especially with a lot of client metadata)
- we place "terminal" items first, and then nested ones below, kind of like a filesystem layout might look in an explorer window, rather than alphabetizing all prefixes. I am less attached to this change but it feels right so far.